controllers/common/views/cms-page.njk

```diff
@@ -0,0 +1,119 @@
+{% from "components/hero.njk" import hero with context %}
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+{% from "components/content-box/macro.njk" import contentBox %}
+{% from "components/segment-links/macro.njk" import segmentLinks %}
+{% from "components/segment/macro.njk" import segment %}
+{% from "components/flexible-content/macro.njk" import flexibleContentPart %}
+{% from "components/miniature-hero/macro.njk" import miniatureHero %}
+{% from "components/section-links/macro.njk" import sectionLinks %}
+
+{% extends "layouts/main.njk" %}
+{% if not pageHero.image %}{% set bodyClass = 'has-static-header' %}{% endif %}
+
+{% set flexAnchor = 'item' %}
+
+{% block content %}
+
+    {% if pageHero.image %}
+        {{ hero(title, pageHero.image) }}
+    {% endif %}
+
+    <main role="main" id="content" {% if pageHero.image %}class="nudge-up"{% endif %}>
+        {% block contentPrimary %}
+
+            {% if content.introduction %}
```

Comment:
All of this is to allow backward compatibility with "legacy" information/listing pages (eg. [this section](https://www.tnlcommunityfund.org.uk/funding/funding-guidance/managing-your-funding)).

Reply:
Is there a way we could work out if it's "legacy" earlier on in the controller and render two different templates?

Reply:
Probably – will have a look.

Reply:
Not a big deal, might make things easier to delete but the logic isn't complicated